index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+  <link rel="stylesheet" href="index.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="https://use.typekit.net/lck4hzp.css">
+  <title>Summit</title>
+</head>
+<body>
+  <header class="header">
+    <div class="social-icons">
+      <a href="https://www.facebook.com/" class="nav-1"><img src="images/fb-icon.svg" alt="fb-icon"></a>
+      <a href="https://twitter.com/" class="nav-1"><img src="images/twitter.svg" alt="twitter-icon"></a>
+      <p class="nav-1">English</p>
+      <p class="nav-1">My page</p>
+      <p class="nav-1">Logout</p>
+    </div>
+    <div class="nav-bar">
+      <div class="logo">
+        <i class="fa-sharp fa-solid fa-bars" id="hamburg-open"></i>
+        <a href="index.html"><img src="images/microsoft.png" alt="logo-icon" class="logo-icon"></a>
+      </div>
+      <nav class="menu-bar">
+        <ul class="nav-container">
+              <li class="item-2"><i class="fa-sharp fa-solid fa-xmark" id="close-btn"></i></li>
+              <li class="item-1"><a href="about.html" class="reference ref-1">About</a></li>
+              <li class="item-1"><a href="#" class="reference">Program</a></li>
+              <li class="item-1"><a href="#" class="reference">Join</a></li>
+              <li class="item-1"><a href="#" class="reference">Sponsor</a></li>
+              <li class="item-1"><a href="#" class="reference">News</a></li>
+          </ul>
+      </nav>
+    </div>
+  </header>
+  <main class="main">
+    <section class="section-1">
+      <div class="cover-div">
+        <div class="into-text">
+          <p class="hello-text">"Hello! Sharing world"</p>
+          <h1 class="head-text">
+              DEVS CONFERENCE<br> 
+              GLOBAL SUMMIT<br> 
+              2023
+          </h1>
+        </div>
+        <div class="sec-text">
+          <p class="overview-text">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Kenya.</p>
+        </div>
+        <div class="date-text">
+          <h3 class="head-date-text">2023.10.16(THU)~16(FRI)</h3>
+          <p class="minor-date-text">@ Microsoft offices of Kenya, Tech Centre and more</p>
```

Comment:
Kindly obey the header hierarchy rule for accessibility's sake😉. You have done a nice job using an `h1` tag for the main header👍. the next hierarchy should be a `h2`  and then, each section should start with a `h2` tag and the subsequent header will use the next hierarchy. `h3`, `h4` etc. shows the hierarchy of the headers (which is under which😉).
ensure this is implemented in your home and about page👍.